constructor/nsis/main.nsi.tmpl

```diff
@@ -37,6 +37,9 @@ Unicode "true"
 !insertmacro GetParameters
 !insertmacro GetOptions
 
+!include "StrFunc.nsh"
+${StrCase}
```

Comment:
Are you asking why I'm using `StrCase` or why it's on line 41?

Reply:
Both I guess 😬

Reply:
I added a few comments explaining this. Functions need to be called like this before they can be used, but that seems to be a quirk of `StrFunc.nsh`: https://nsis.sourceforge.io/StrFunc#3.1_How_To_Use_Commands_In_Install_Sections_and_Functions

I use it because tests require the `::error::` string to be lower case whereas the error messages that users are seeing should be sentence case. Using `StrCase` makes the code DRY-er.

Reply:
Would it be better to instead test strings in lowercase / case-insensitive expressions?

Reply:
In general, yes. Another thing to note though: all other instances of `DetailPrint` use `::error::`. If we didn't use `StrCase` here, this would be the only instance where it's `::Error::`.

Reply:
Then let's do that instead, looks cleaner! I'd argue we merge #675 first so we can add that change here. If you have some time to do review it quickly, then we are good to go!

Reply:
I went ahead and made the changes. #675 looks orthogonal enough that we don't need to merge it first, I think